ios/RNSScreen.m

```diff
@@ -58,9 +58,10 @@ - (void)updateBounds
   [_bridge.uiManager setSize:self.bounds.size forView:self];
 }
 
-- (void)setActivityState:(int)activityState
+- (void)setActivityStateOrNil:(NSNumber *)activityStateOrNil
```

Comment:
Would be great to document it somewhere in the code the purpose of special handling nil value and also explain when nil can be provided. That is we should mention that `nil` will be provided when `activityState` it is set as an animated value and we change it from JS to be a plain value (non animated). I understand that in this scenario we first trigger the code which resets it to the default, which is nil, and which maps to 0 meaning Inactive. In that case we want to ignore such update as soon after we get the actual state passed from JS.